SU-SWS/ansible-playbooks
# SITES-1169: Migrate SWS sites to ACE

# READY FOR REVIEW

# Summary
- Correct short names of acquia_connector modules
- Remove stanfordpro.aliases.drushrc.php from version control

# Needed By (Date)
- Soon

# Criticality
- How critical is this PR on a 1-10 scale? 10/10
- All sites stall out at the `drush -y en acquia_connector ...` step, thus we are prevented from migrating anything to StanfordPro

# Steps to Test

1. Check out this branch
2. Copy the the `stanfordpro.aliases.drushrc.php` file from https://github.com/SU-SWS/ansible-sync/pull/10 into `roles/setup-local/templates`.
3. Migrate any site to any environment
4. Succeed

# Affected Projects or Products
- StanfordPro migrations

# Associated Issues and/or People
- JIRA ticket: SITES-1169
- Other PRs: https://github.com/SU-SWS/ansible-sync/pull/10

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
